cni/network/network_test.go

```diff
@@ -1668,6 +1669,107 @@ func TestPluginSwiftV2MultipleAddDelete(t *testing.T) {
 	}
 }
 
+func TestPluginGenerateEndpointNames(t *testing.T) {
```

Comment:
unit test doesn't feel right to me.. unit test need to validate if right ifname passed to createendpoint.. lets skip adding ut for now.. can figure out later for this case.

Reply:
there is a findMasterInterface() that we cannot mock because it calls wireserver to find windows interfaces; so in my UT, I just mock:
1.Create interfaceInfos and createEpInfo
2.Test if condition for epInfoOpt.ifInfo.NICType == cns.InfraNIC && !*epInfoOpt.infraSeen else..
3.From here we get the ifName and endpointID(endpointName)
4.Then check if endpointID is correct and expected comparing with containerID where we get from plugin.nm.GetEndpointID()